prime-react/src/main/scala/lucuma/react/primereact/DropdownBase.scala

```diff
@@ -65,6 +72,20 @@ object DropdownBase {
       .applyOrNot(props.disabled, _.disabled(_))
       .applyOrNot(props.dropdownIcon, _.dropdownIcon(_))
       .applyOrNot(props.tooltip, _.tooltip(_))
+      .applyOrNot(
+        props.itemTemplate,
+        (c, p) =>
+          c.itemTemplate(raw =>
+            p(props.selectItemFinder(raw.asInstanceOf[CSelectItem].value)).rawNode
```

Comment:
I'm guessing this is to deal with unfriendly types from ST.

Reply:
Yes